boolean/nand/a5f368aa-2cea-4110-a170-c08ec18a6a63.json

```diff
@@ -4,18 +4,21 @@
   "interface": {
     "inputs": {
       "ad9e2497-ee64-46bc-ae85-86ce71a33bd9": {
-        "type": "boolean\n",
-        "name": "a"
+        "type": "boolean",
+        "name": "a",
+        "description": "Must be a boolean eg true or false"
```

Comment:
I'd avoid describing the type of the port in the description as the port has a type field for that.
We'd rather try to describe its purpose and if needed an example

For coherence with the other boolean nodes (like `boolean/And` I suggest this simple description:
"Receives the first operand"

Reply:
updated the description as you have suggested :)